data/wikipron/whitelist/eng_us_phonemic.whitelist

```diff
@@ -0,0 +1,66 @@
+ɪ
+ə
+t
+n
+s
+ɹ
+l
+k
+d
+m
+i
+æ
+p
+ɛ
+b
+ʊ
+e
+a
+z
+f
+ʃ
+ɑ
+o
+ɡ
+v
+ʌ
+iː
+ŋ
+ɚ
+j
+ɔ
+w
+ʒ
+h
+uː
+ɒ
+u
+ɑː
+θ
+ɝ
+ɔː
+d͡ʒ
+l̩
+t͡ʃ
+n̩
+ð
+ɨ
+ɜː
+r
+ɜ
+ɪ̯
+ɾ
+m̩
+ʍ
+ʊ̯
+ɝː
+x
```

Comment:
This, the voiceless velar fricative, is not found in any native English word (at least since the modern era) and is a good example of a non-native phoneme.